Import WPT /css/css-view-transitions tests

tuankiet65 opened the pull request:
#### 3d803396a4a24ea31a6a8dac02633a92fec45168
<pre>
Import WPT /css/css-view-transitions tests
<a href="https://rdar.apple.com/133727010">rdar://133727010</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=277999">https://bugs.webkit.org/show_bug.cgi?id=277999</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/67d7971c98261d9f90697ee8eb83c0dc11452ff4">https://github.com/web-platform-tests/wpt/commit/67d7971c98261d9f90697ee8eb83c0dc11452ff4</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-on-non-root-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-on-non-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-type-on-non-root-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-type-on-non-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/block-with-overflowing-text-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/block-with-overflowing-text-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/block-with-overflowing-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/clip-path-larger-than-border-box-on-child-of-named-element.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/fractional-translation-from-position-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/fractional-translation-from-position-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/fractional-translation-from-position.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/multiline-span-with-overflowing-text-and-box-decorations-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/multiline-span-with-overflowing-text-and-box-decorations-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/multiline-span-with-overflowing-text-and-box-decorations.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pagereveal-ctor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pagereveal-ctor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pageswap-ctor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pageswap-ctor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform-with-scale.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform-with-scale.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform.tenative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform.tenative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-direct.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-direct.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested-vt-names.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested-vt-names.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-ancestor.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-ancestor.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-existent.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-existent.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-self.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-self.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-direct.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-direct.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-nested.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-nested.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-on-self.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-on-self.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-direct.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-direct.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-ignores-nearest-name.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-ignores-nearest-name.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-nested.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-nested.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-exit.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-exit.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-display-none.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-display-none.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-in-pseudo-basic.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-in-pseudo-basic.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-ref-100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/normal-goes-up.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/normal-goes-up.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/render-element.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/render-element.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/resources/compute-common.css: Added.
(::view-transition,):
(html.no-match::view-transition):
(.green):
(.test):
(.green-ref):
(.red):
(.test-ref):
(.red-ref):
(.nearest-ref):
(.normal):
(.normal-ref):
(.contain-ref):
(::view-transition-group(green)):
(::view-transition-group(test)):
(::view-transition-image-pair(*),):
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/resources/compute-test.js: Added.
(add_rule):
(mode.string_appeared_here.onload):
(else):
(else.onload.async const):
(else.onload):
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-spans-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-spans-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-spans.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-is-inline-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-is-inline-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-is-inline.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-named-elements-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-named-elements-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-named-elements.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/pseudo-elements-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/pseudo-elements-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-invalid.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-invalid.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-valid.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-valid.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-entry.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-exit.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-ident.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-multiple-wildcard.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-multiple.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-wildcard-no-star.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-wildcard.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-mismatch-ident.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-mismatch-partial.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-mismatch-wildcard.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-multiple-vt-classes.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-new-with-class-old-without.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-old-with-class-new-without.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-view-transition-group.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-view-transition-image-pair.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow-important-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow-important.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-and-box-decorations-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-and-box-decorations-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-and-box-decorations.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/view-transition-types-matches.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/w3c-import.log:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/282231@main">https://commits.webkit.org/282231@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c5aec5eb0a872486d530c227aa4f386990279982

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/62528 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/41883 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/15123 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/66512 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/13080 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/49570 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/13416 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/50380 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/9014 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/65597 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/38914 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/54151 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/31123 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/35630 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/11477 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/12008 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/57237 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/11793 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/68241 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/6474 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/11492 "Found 1 new test failure: workers/worker-user-gesture.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/57756 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/6503 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/54189 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/57948 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/5392 "Passed tests") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/9410 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/37683 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/38768 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/39865 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/38512 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/platform/mac/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-exit.tentative_vtmode=crossdoc-expected.png:
Please don't generate these and skip the test instead

tuankiet65 replied:
Removed.

nt1m commented on LayoutTests/platform/mac/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-exit.tentative_vtmode=crossdoc-expected.txt:
ditto

tuankiet65 replied:
Removed.

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/view-transition-types-matches.html:
```diff
@@ -31,6 +30,7 @@
 html:active-view-transition-type(foo,) #negative5 { background: red; }
 html:active-view-transition-type() #negative6 { background: red; }
 html:active-view-transition-type(,) #negative7 { background: red; }
+html:has(:active-view-transition-type(bar)) #negative8 { background: green; }
```
This needs to change to `red` upstream.

tuankiet65 replied:
This is done.

tuankiet65 committed:
Import WPT /css/css-view-transitions tests
rdar://133727010
https://bugs.webkit.org/show_bug.cgi?id=277999

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/67d7971c98261d9f90697ee8eb83c0dc11452ff4

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-on-non-root-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-on-non-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-type-on-non-root-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-type-on-non-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/block-with-overflowing-text-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/block-with-overflowing-text-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/block-with-overflowing-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/clip-path-larger-than-border-box-on-child-of-named-element.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/fractional-translation-from-position-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/fractional-translation-from-position-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/fractional-translation-from-position.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-below-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-left-of-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-on-top-of-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-new-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-new.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-old-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-old.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/massive-element-right-of-viewport-partially-onscreen-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/multiline-span-with-overflowing-text-and-box-decorations-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/multiline-span-with-overflowing-text-and-box-decorations-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/multiline-span-with-overflowing-text-and-box-decorations.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pagereveal-ctor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pagereveal-ctor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pageswap-ctor-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/pageswap-ctor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform-with-scale.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform-with-scale.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform.tenative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/adjust-transform.tenative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-direct.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-direct.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested-vt-names.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested-vt-names.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-nested.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-ancestor.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-ancestor.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-existent.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-non-existent.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-self.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/compute-explicit-name-self.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-direct.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-direct.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-nested.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-nested.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-on-self.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/contain-on-self.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-direct.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-direct.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-ignores-nearest-name.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-ignores-nearest-name.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-nested.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nearest-nested.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-exit.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-exit.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-display-none.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-display-none.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-in-pseudo-basic.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-group-in-pseudo-basic.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-ref-100.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/nested-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/normal-goes-up.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/normal-goes-up.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/render-element.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/render-element.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/resources/compute-common.css: Added.
(::view-transition,):
(html.no-match::view-transition):
(.green):
(.test):
(.green-ref):
(.red):
(.test-ref):
(.red-ref):
(.nearest-ref):
(.normal):
(.normal-ref):
(.contain-ref):
(::view-transition-group(green)):
(::view-transition-group(test)):
(::view-transition-image-pair(*),):
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/resources/compute-test.js: Added.
(add_rule):
(mode.string_appeared_here.onload):
(else):
(else.onload.async const):
(else.onload):
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/nested/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-spans-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-spans-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-captures-spans.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-is-inline-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-is-inline-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-is-inline.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-named-elements-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-named-elements-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-named-elements.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/pseudo-elements-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/pseudo-elements-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-invalid.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-invalid.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-valid.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/view-transition-group-valid.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-entry.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-exit.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-ident.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-multiple-wildcard.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-multiple.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-wildcard-no-star.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-match-wildcard.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-mismatch-ident.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-mismatch-partial.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-mismatch-wildcard.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-multiple-vt-classes.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-new-with-class-old-without.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-old-with-class-new-without.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-view-transition-group.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/pseudo-with-classes-view-transition-image-pair.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow-important-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/inline-element-size-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow-important.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class-inside-shadow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-class.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-and-box-decorations-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-and-box-decorations-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-and-box-decorations.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/span-with-overflowing-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/view-transition-types-matches.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/w3c-import.log:
* LayoutTests/platform/mac-wk2/TestExpectations:

Canonical link: https://commits.webkit.org/282231@main
